tests/fixtures/models.py

```diff
@@ -35,9 +35,9 @@ def __init__(self, num_channels: int, num_classes: int) -> None:
         # These tests attempt to perform surgery on `fc1` layer, and we want
         # to make sure that post-surgery, self.fc1 refers to the same parameters
         # as self.net[1]
-        self.fc1 = torch.nn.Linear(num_channels, 5)
+        self.fc1 = torch.nn.Linear(num_channels, 20)
```

Comment:
this may slow down other tests -- is this strictly neccessary?

Reply:
Switched back